grpc/rust/build.rs

```diff
@@ -1,43 +0,0 @@
-//
```

Comment:
We actually refactored (moved) this file to the `grpc/rust/compile` package (but Git doesn't know because we also massively changed the code)